.github/workflows/pr-commands.yml

```diff
@@ -19,8 +19,9 @@ jobs:
           ref: ${{ env.DEFAULT_BRANCH }}
       - name: Check if the author is a member or Owner
         id: check-condition
+        env:
+          slash_command: ${{github.event.comment.body}}
```

Comment:
@lociko thanks for taking a look at this, from what I can tell it seems like this isn't really used anywhere I think we can take it out entirely

Reply:
@lociko I understand, it's a valid concern. I am saying, as part of your PR can you make the change to remove this line entirely instead?

Reply:
I misunderstood you, sorry
Done